tests/tap_tester/test_all_fields.py

```diff
@@ -87,6 +87,12 @@ def test_run(self):
                     
                 # verify all fields for each stream are replicated
                 self.assertSetEqual(expected_all_keys, actual_all_keys)
+                
+                # Verify we have more than one app data
+                # below four streams are independent of the app_id
+                if stream not in ["accounts", "visitors", "metadata_accounts", "metadata_visitors"]:
+                    records_appid_set = set([message.get('data').get('app_id') for message in messages.get("messages")])
+                    self.assertGreater(len(records_appid_set), 1, msg=f"We have only one app's records for {stream}")
```

Comment:
This is a good assertion to cover the case when the `app_ids` are not set in the config. Could you add a configurable properties test (at the tap-tester level) to verify the tap can execute a sync when called via the orchestrator with `app_ids` set in the config?

Reply:
Thanks @kspeer825, I have added the integration test ["test_app_ids"](https://github.com/singer-io/tap-pendo/blob/TDL-17610-multiple-apps-support-and-data-fetching/tests/tap_tester/test_app_ids.py) as you suggested.